src/gui/touchscreengui.cpp

```diff
@@ -886,7 +892,11 @@ void TouchScreenGUI::translateEvent(const SEvent &event)
 					m_move_id                  = event.TouchInput.ID;
 					m_move_has_really_moved    = false;
 					m_move_downtime            = porting::getTimeMs();
-					m_move_downlocation        = v2s32(event.TouchInput.X, event.TouchInput.Y);
+					if (m_use_crosshair || m_camera_mode == CAMERA_MODE_THIRD) {
+						m_move_downlocation = v2s32(m_screensize.X / 2, m_screensize.Y / 2);
+					} else {
+						m_move_downlocation = v2s32(event.TouchInput.X, event.TouchInput.Y);
+					}
```

Comment:
Moving the two if-else blocks out of the way of these aligned lines would make the code look nicer. :)

Reply:
Done.